.gitignore

```diff
@@ -7,3 +7,4 @@ __pycache__
 /src/pypi_changes/_version.py
 /src/*.egg-info
 /.*_cache
+/.venv
```

Comment:
IMHO this is out of scope for this gitignore. Such configuration should be in your global git config and not in each individual project.

Reply:
Ah I didn't know about global git ignores, that's a great tip.

Reply:
Done